react-native-gutenberg-bridge/ios/GutenbergBridgeDelegate.swift

```diff
@@ -11,20 +11,23 @@ public struct MediaInfo {
 }
 
 public typealias MediaPickerDidPickMediaCallback = (_ media: [MediaInfo]?) -> Void
-
 public typealias MediaImportCallback = (_ media: MediaInfo?) -> Void
 
-public enum MediaPickerSource: String {
-    case mediaLibrary = "SITE_MEDIA_LIBRARY"
-    case deviceLibrary = "DEVICE_MEDIA_LIBRARY"
-    case deviceCamera = "DEVICE_CAMERA"
-}
-
-public enum MediaFilter: String {
-    case image
-    case video
-    case audio
-    case other
```

Comment:
Merged `MediaType` with `MediaFilter` since we filters by `Types` of media. It seemed like redundant.